src/cljs/reagent_toolbox_docs/layout.cljs

```diff
@@ -42,7 +42,18 @@
                 ["Autocomplete" :autocomplete]
                 ["Avatar" :avatar]
                 ["Button" :button]
+                ["Checkbox" :checkbox]
+                ["Drawer" :drawer]
+                ["Dropdown" :dropdown]
+                ["Font Icon" :font-icon]
                 ["Input" :input]
+                ["Link" :link]
+                ["Navigation" :navigation]
+                ["Progress Bar" :progress-bar]
+                ["Slider" :slider]
+                ["Snackbar" :snackbar]
+                ["Switch" :reagent-switch]
+                ["Radio Buttons" :radio-buttons]
```

Comment:
As soon as I learn how to read... Wait, no, done that already. Just a simple mistake here.